src/Uno.UI.FluentTheme/Resources/PriorityDefault/FlipView_themeresources.xaml

```diff
@@ -76,7 +76,7 @@
     </ResourceDictionary.ThemeDictionaries>
 
     <!-- UNO TODO Currently only style using native flip view control is supported -->
-    <!--<Style TargetType="FlipView" BasedOn="{StaticResource DefaultFlipViewStyle}" />-->
+    <Style TargetType="FlipView" BasedOn="{StaticResource DefaultFlipViewStyle}" />
```

Comment:
Shouldn't this be restricted to certain platforms? We want to continue to use the native style for Android and iOS.

Reply:
Updated. Added filters so that only `SKIA` and `WASM` use this style